engine/runtime/src/main/java/org/enso/interpreter/runtime/Module.java

```diff
@@ -109,6 +113,7 @@ public Module(QualifiedName name, Package<TruffleFile> pkg, TruffleFile sourceFi
     this.cache = new ModuleCache(this);
     this.wasLoadedFromCache = false;
     this.hasCrossModuleLinks = false;
+    this.virtual = false;
```

Comment:
I think you said you prefer `synthetic` in some discord discussion? I'm with you, it's probably a better name, given `virtual` means something very different in PLD.

Reply:
To paraphrase the known quote, naming things is one of the two hard things in CS....
I will make the change. Thanks for taking a look!